src/reaper_osara.cpp

```diff
@@ -4810,6 +4811,62 @@ void cmdInsertMarker(Command* command) {
 	outputMessage(format(translate("marker {} inserted"), number));
 }
 
+void cmdInsertProjectMarker(Command* command) {
+	cmdhInsertProjectMarker(command->gaccel.accel.cmd);
+}
+
+void cmdhInsertTakeMarker(int command) {
+	if(CountSelectedMediaItems(0)>0) {
```

Comment:
If there are no selected items, this means we will never pass the command to REAPER. Practically, this should be fine, since REAPER will probably do nothing in the case of 0 selected items. However, when intercepting commands, I do think we should always pass them to REAPER, on the very slight chance that REAPER does something we don't expect, either now or in future. Otherwise, the action will just do nothing, even if it would have done something without OSARA installed.

You could either:

1. Check if the item count is 0, call Main_OnCommand and return; or
2. Don't check at all and rely on the fact that 0 items means the loops will do nothing.